pyo3-derive-backend/src/from_pyobject.rs

```diff
@@ -133,7 +133,9 @@ impl<'a> Container<'a> {
                 "Cannot derive FromPyObject for empty structs and variants.",
             ));
         }
-        let transparent = attrs.iter().any(ContainerAttribute::transparent);
+        let transparent = attrs
```

Comment:
Changes here are unrelated?

Reply:
I did for address clippy warnings.

Reply:
Although clippy only asked to replace the implementation with `matches!()`, no?

Reply:
Yeah, but since we support MSRV, we cannot use `macthes`.